cmd/utils.go

```diff
@@ -39,7 +42,15 @@ func configure(ctx *cli.Context) (c *tfcw.Client, cfg *schemas.Config, err error
 
 	tfcwConfigFile := computeConfigFilePath(cfg.Runtime.WorkingDir, ctx.GlobalString("config-file"))
 	log.Debugf("Using config file at %s", tfcwConfigFile)
-	err = hclsimple.DecodeFile(tfcwConfigFile, nil, cfg)
+
+	// Create and EvalContext to define functions that we can use within the HCL for interpolation
+	eval_ctx := &hcl.EvalContext{
```

Comment:
I reckon the linter is not happy about the snake case

Reply:
Ah yes! fixed!